pages/study/timer/index.js

```diff
@@ -48,8 +40,32 @@ export default function Timer() {
             ))}
           </select>
         </div>
-
       </div>
     </div>
   )
+
+  function showTimerFields(timerTotalSeconds) {
+    let minutes = Math.floor(timerTotalSeconds / 60);
+    let seconds = timerTotalSeconds - minutes * 60;
+
+    minutes = minutes <= 9 ? '0' + minutes : minutes;
+    seconds = seconds <= 9 ? '0' + seconds : seconds;
+    return <p>{minutes}:{seconds}</p>
+  }
+
+  function showStartButton(typeInterval) {
+    if(typeInterval === 'Group\'s Timer'){
+      return;
+    }
+
+    return <button onClick={() => {
+      setTimerTotalSeconds(typeInterval * 60 - 1);
+    }} className={styles.pushable}>
```

Comment:
Maybe rename this one to something closer to the action it's tied to. Once code gets bigger we will probably start using joint classes for multiple elements therefore if their name equals closer to the action they are related to it's going to be easier to track down and patch potential future issues.